packages/components/src/components/primitives/Chat/Conversation/MediaMessage/MediaMessage.tsx

```diff
@@ -0,0 +1,75 @@
+import React from 'react';
+import cn from 'classnames';
+import { Image } from 'components/primitives/Image';
+import { Image as ImageType } from "types";
+import {
+  Typography,
+  TypographyTypes,
+  TypographyMode,
+} from 'components/primitives/Typography';
+import { getStyles } from './MediaMessage.styles';
+
+export const WRAPPER_MEDIA_MESSAGE_DEFAULT_TEST_ID = 'wrapperMediaMessage';
+
+export enum MediaMessageBorderColor {
+  Black10 = 'border-fill-black-10',
+  Primary25 = 'border-fill-primary-25',
+  Secondary25 = 'border-fill-secondary-25',
+}
+
+export enum MessagePlacement {
+  Left = 'left',
+  Right = 'right',
+}
+
+type Props = {
+  className?: string;
+  borderColor?: MediaMessageBorderColor;
+  time?: string;
+  placement?: MessagePlacement;
+  image: ImageType;
+  onClick?: () => void;
+  testId?: string;
+}
```

Comment:
Yesterday we mentioned that this component should be capable to render media + regular message (?) Am I right? Or do we want to create a wrapper on top of this component? @tkanzakic